### ISSUE TITLE
rstan 2.8.1 does not report "numerical problems"

### ISSUE THREAD

**Issue by johnbaums** [stan-dev/rstan#232]
`rstan 2.8.1`, run from RStudio Version 0.99.467 on Windows, shows model fitting progress in the Viewer pane. On completion of sampling (using `stanc`), I receive warnings, e.g. `[1] "The following numerical problems occured the indicated number of times after warmup on chain 1"`, but no "problems" are listed.

It's possible that the problems are listed once all chains have completed sampling, but I'm fitting multiple models, and the Viewer pane clears when a new model starts so I lose this info (is there a way to have progress printed to the console instead, I wonder?).

```
Click the Refresh button to see progress of the chains
starting worker pid=14764 on localhost:11213 at 11:14:24.102
starting worker pid=12912 on localhost:11213 at 11:14:24.369
starting worker pid=7568 on localhost:11213 at 11:14:24.597

SAMPLING FOR MODEL '5e1a2ea3d58ddaea072bcad9cc158896' NOW (CHAIN 1).

Chain 1, Iteration:    1 / 2000 [  0%]  (Warmup)
SAMPLING FOR MODEL '5e1a2ea3d58ddaea072bcad9cc158896' NOW (CHAIN 2).

Chain 2, Iteration:    1 / 2000 [  0%]  (Warmup)
SAMPLING FOR MODEL '5e1a2ea3d58ddaea072bcad9cc158896' NOW (CHAIN 3).

Chain 3, Iteration:    1 / 2000 [  0%]  (Warmup)
Chain 3, Iteration:  200 / 2000 [ 10%]  (Warmup)
Chain 2, Iteration:  200 / 2000 [ 10%]  (Warmup)
Chain 1, Iteration:  200 / 2000 [ 10%]  (Warmup)
Chain 1, Iteration:  400 / 2000 [ 20%]  (Warmup)
Chain 1, Iteration:  600 / 2000 [ 30%]  (Warmup)
Chain 1, Iteration:  800 / 2000 [ 40%]  (Warmup)
Chain 1, Iteration: 1000 / 2000 [ 50%]  (Warmup)
Chain 1, Iteration: 1001 / 2000 [ 50%]  (Sampling)
Chain 1, Iteration: 1200 / 2000 [ 60%]  (Sampling)
Chain 1, Iteration: 1400 / 2000 [ 70%]  (Sampling)
Chain 1, Iteration: 1600 / 2000 [ 80%]  (Sampling)
Chain 1, Iteration: 1800 / 2000 [ 90%]  (Sampling)
Chain 1, Iteration: 2000 / 2000 [100%]  (Sampling)
#  Elapsed Time: 1395.64 seconds (Warm-up)
#                422.031 seconds (Sampling)
#                1817.67 seconds (Total)

[1] "The following numerical problems occured the indicated number of times after warmup on chain 1"

Chain 3, Iteration:  400 / 2000 [ 20%]  (Warmup)
Chain 2, Iteration:  400 / 2000 [ 20%]  (Warmup)
Chain 2, Iteration:  600 / 2000 [ 30%]  (Warmup)
Chain 2, Iteration:  800 / 2000 [ 40%]  (Warmup)
Chain 2, Iteration: 1000 / 2000 [ 50%]  (Warmup)
Chain 2, Iteration: 1001 / 2000 [ 50%]  (Sampling)
Chain 2, Iteration: 1200 / 2000 [ 60%]  (Sampling)
Chain 2, Iteration: 1400 / 2000 [ 70%]  (Sampling)
Chain 2, Iteration: 1600 / 2000 [ 80%]  (Sampling)
Chain 2, Iteration: 1800 / 2000 [ 90%]  (Sampling)
Chain 2, Iteration: 2000 / 2000 [100%]  (Sampling)
#  Elapsed Time: 3176.87 seconds (Warm-up)
#                229.266 seconds (Sampling)
#                3406.13 seconds (Total)

[1] "The following numerical problems occured the indicated number of times after warmup on chain 2"

Chain 3, Iteration:  600 / 2000 [ 30%]  (Warmup)
```

**Comment by bgoodri** [comment]
Thanks. I'll fix this so that the reported "numerical problems" actually appear. But on Windows it is basically impossible (AFAIK and we've tried) to run the chains in parallel and get to progress and messages from the child process to appear in the Console. So, each chain writes to the same file in `tempdir` and that is what is redirected to the Viewer pane in RStudio on Windows.

**Comment by johnbaums** [comment]
Thanks Ben, that makes sense. I'm interested to see how you achieved it (progress/messages for parallel processes) for other OSes - I'll dig around the source. Cheers.

**Comment by bgoodri** [comment]
Look for the string open_progress in R/stanmodel-class.R

**Comment by bgoodri** [comment]
This was fixed for 2.8.2
